src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -217,6 +234,10 @@ public String getRepositoryName() {
         return getBitbucketSCMRepository().getRepositoryName();
     }
 
+    public String getMirrorName() {
+        return getBitbucketSCMRepository().getMirrorName();
```

Comment:
`BitbucketSCMRepository.getMirrorName()` is marked as `@Nullable`. Either this `getMirrorName` should also be `Nullable or we should handle the case where its null (maybe strip to empty?)

Reply:
It is never null. Its either empty (Primary server) or has a value. I have fixed the annotation